allennlp/data/tokenizers/pretrained_transformer_tokenizer.py

```diff
@@ -63,16 +63,17 @@ def __init__(
         stride: int = 0,
         truncation_strategy: str = "longest_first",
         calculate_character_offsets: bool = False,
+        **tokenizer_kwargs,
```

Comment:
This should be `tokenizer_kwargs: Dict[str, Any] = None`.  The reason for that would be apparent if you tried to construct this using `Tokenizer.from_params`.  Actually, adding a test for that would be good; if it works using this way of doing things, I might be ok with keeping it like it is.  I really don't think it'll work, though.  It'll tell you "extra parameters passed".

Reply:
Done